ztp/Makefile

```diff
@@ -4,6 +4,8 @@ export GOFLAGS := $(if $(GOFLAGS),$(GOFLAGS),-mod=vendor)
 
 .PHONY: ci-job govet gotest
 ci-job: govet gotest
+.PHONY: ci-job govet gotest checkSourceCrs
+ci-job: govet gotest checkSourceCrs
```

Comment:
The diff makes it look like the .PHONY and ci-job lines are duplicated rather than changed. Can you verify this?